MAUI/ImageEditor/getting-started.md

```diff
@@ -15,11 +15,11 @@ This section explains the steps to create and load an image to the image editor
 

```

Comment:
<img width="351" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/92788387/052927eb-39d4-424e-a287-e986005d3a3a">

This section explains the steps to create and load an image to the `.Net MAUI ImageEditor(SfImageEditor)`(https://www.syncfusion.com/maui-controls/maui-image-editor)
 control.